Index: main
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"Hello World\")\nTesting Fetch\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main b/main
--- a/main	(revision 74b7958448a75f738f76e0fd71012bce0129eff0)
+++ b/main	(date 1748360961979)
@@ -1,2 +1,1 @@
 print("Hello World")
-Testing Fetch
Index: testAPI.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/testAPI.py b/testAPI.py
--- a/testAPI.py	(revision 74b7958448a75f738f76e0fd71012bce0129eff0)
+++ b/testAPI.py	(date 1748360961984)
@@ -1,1 +1,0 @@
-
