Index: main
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"Hello World\")\nTesting Fetch\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main b/main
--- a/main	(revision 2e875324b58289440385c43ad8a0fa436fca46b0)
+++ b/main	(date 1748360527225)
@@ -1,2 +1,1 @@
 print("Hello World")
-Testing Fetch
Index: testAPI.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/testAPI.py b/testAPI.py
--- a/testAPI.py	(revision 2e875324b58289440385c43ad8a0fa436fca46b0)
+++ b/testAPI.py	(date 1748360743113)
@@ -1,1 +1,0 @@
-
